docs/admin/projects.rst

```diff
@@ -764,30 +763,29 @@ By default the component is visible to anybody who has access to the project,
 even if the person can not perform any changes in the component. This makes it
 easier to keep translation consistency within the project.
 
-Enable this in case you want to grant access to this component explicitly -
-the project level permissions will not apply and you will have to specify
-component or component list level permission in order to grant access.
+Turning this on at component-, or component-list level takes over
+access permission to a component, regardless of project-level permissions.
```

Comment:
```suggestion
access permission to a component, regardless of project-level permissions, and you will have to grant access to it explicitly. This can be done through making a group that has access and granting it to users by putting them in it, or using the default `custom` or `private` access control groups.
```
@Fat-Zer Something like this? It could be moved to the access control docs, but good for now?